src/textual/reactive.py

```diff
@@ -177,8 +177,8 @@ def __set__(self, obj: Reactable, value: ReactiveType) -> None:
         validate_function = getattr(obj, f"validate_{name}", None)
         # Check if this is the first time setting the value
         first_set = getattr(obj, f"__first_set_{self.internal_name}", True)
-        # Call validate, but not on first set.
-        if callable(validate_function) and not first_set:
```

Comment:
Do you remember the reasoning for this? The comment and the code suggest this behaviour was intentional, but I'm struggling to think of why that might be.

Reply:
I think it was a workaround to avoid an issue with the default value invoking validators before the DOM was set up. Maybe prior to `init`. Could very well be deprecated.